bindings/rust/README.md

```diff
@@ -11,21 +11,27 @@ In order to generate rust bindings for s2n-tls, you need to have the following i
 * libclang - this is usually installed through your system's package manager
 * libssl-dev
 * pkg-config
-
 ## Usage
 
 Generating rust bindings can be accomplished by running the `generate.sh` script:
 
 ```
-$ ./bindings/rust/generate.sh
+$ ./bindings/rust/extended/generate.sh
 ```
 
 This script generates the low-level bindings in the crate `s2n-tls-sys`, which is used by the `s2n-tls` crate to provide higher-level bindings.
 See [s2n-tls-sys](https://github.com/aws/s2n-tls/blob/main/bindings/rust/s2n-tls-sys/README.md) for more information on `s2n-tls-sys` crate.
 
 ## Minimum Supported Rust Version (MSRV)
 
-`s2n-tls` will maintain a rolling MSRV (minimum supported rust version) policy of at least 6 months. The current s2n-quic version is not guaranteed to build on Rust versions earlier than the MSRV.
+There are two rust bindings workspaces that have different MSRV policies. Crates in `standard` maintain a rolling MSRV policy of at least 6 months. Crates in `extended` maintain an older MSRV for increased support.
+
+### Extended
+
+The current MSRV for `s2n-tls`, `s2n-tls-sys` and `s2n-tls-tokio` is [1.63.0](https://releases.rs/docs/1.63.0/).
+
+### Standard
 
-The current MSRV is [1.63.0][msrv-url].
+We will maintain a rolling MSRV (minimum supported rust version) policy of at least 6 months. The current s2n-quic version is not guaranteed to build on Rust versions earlier than the MSRV.
 
+The current MSRV for the standard workspace is [1.74.0](https://releases.rs/docs/1.74.0/)
```

Comment:
```suggestion
```

In this case we probably don't want to explicitly mention the MSRV, because we'll either need to update it every time we bump the MSRV or it's going to be inaccurate.